adapters/rubicon/rubicon.go

```diff
@@ -53,7 +53,7 @@ type rubiconImpExtRPTrack struct {
 
```

Comment:
Are `Inventory` and `Visitor` these two not necessary to omit if empty?
```
 40 type rubiconParams struct {
 41     AccountId int                `json:"accountId"`
 42     SiteId    int                `json:"siteId"`
 43     ZoneId    int                `json:"zoneId"`
 44     Inventory json.RawMessage    `json:"inventory"`
 45     Visitor   json.RawMessage    `json:"visitor"`
 46     Video     rubiconVideoParams `json:"video"`
 47 }
 48
adapters/rubicon/rubicon.go
```

Reply:
Forgot a few lines. Thank you.